README.md

```diff
@@ -433,7 +433,7 @@ There are some special cases:
 
    singleton constructr: `SCons`
 
-   symbols: `ConsSym0`, `ConsSym1`
+   symbols: `:@#$%^%$#@$`, `:@#$%^%$#@$`
```

Comment:
Note: I didn't change conventions here! I checked, and it turns out the defunctionalization symbols for `:` weren't actually named `ConsSym0` and `ConsSym1` at all, but rather `(:$)` and `(:$$)`. I just opted to update this reference to be correct here, since I happened to change what `(:$)` and `(:$$)` were named.